compiler/cpp/src/thrift/generate/t_java_generator.cc

```diff
@@ -3328,6 +3328,7 @@ void t_java_generator::generate_service_client(t_service* tservice) {
       // Careful, only return _result if not a void function
       if (!(*f_iter)->get_returntype()->is_void()) {
         f_service_ << indent() << "if (result." << generate_isset_check("success") << ") {" << '\n'
+                   << indent() << "  after();" << '\n'
```

Comment:
Wouldn't you want this to be called even if it is a void function? Adding this call here only calls it when it isn't a void function.

Reply:
My previous idea was that it would only be called when the data is returned

Reply:
If the contents of the byte array are confirmed not to be used again after this method is called, it can be called directly